Extension/YapoSubModule.cs

```diff
@@ -30,8 +30,7 @@ protected override void OnSubModuleLoad()
 
                 UIExtender.Register();
                 new Harmony("YAPO").PatchAll();
-            }
-            catch (Exception exception)
+            } catch (Exception exception)
```

Comment:
Shouldn't this stay on a new line?

Reply:
I have the format settings to put catch, finally, else, else if etc on the same line as the closing brace, reduces line count and improves readability imo

Reply:
If you want to keep it then it should be done consistently. I.e. curly braces on the same line as classes, methods, if-else, for, while, try-catch etc. Not only for try-catch and not only when its your formatter.